ports/megamanx816-bit/port.json

```diff
@@ -1,28 +1,28 @@
 {
-  "version": 3,
-  "name": "megamanx816-bit.zip",
-  "items": [
-    "Mega Man X8 16-bit.sh",
-    "megamanx816-bit"
-  ],
-  "items_opt": [],
-  "attr": {
-    "title": "Megaman X8 16-bit",
-    "porter": [
-      "Damian2809"
+    "version": 3,
+    "name": "megamanx816-bit.zip",
+    "items": [
+        "Mega Man X8 16-bit.sh",
+        "megamanx816-bit"
     ],
-    "desc": "The Mega Man X8 16-Bit Demake is a fan-made project that reimagines the original Mega Man X8 in the style of the classic 16-bit era. This demake brings the modern aesthetics and mechanics of the original game into a retro environment reminiscent of the SNES era, particularly the earlier Mega Man X titles on that system.",
-    "desc_md": null,
-    "inst": "https://sonicfangameshq.com/forums/showcase/mega-man-x8-16-bit-demo3.1504/ grab the zip from here, extract it and rename the File to \"Megaman X8 FC.pck\" after that put it in megamanx816-bit/gamedata folder",
-    "inst_md": null,
-    "genres": [
-      "platformer"
-    ],
-    "image": null,
-    "rtr": false,
-    "exp": false,
-    "runtime": "frt_3.5.2.squashfs",
-    "reqs": [],
-    "arch": []
-  }
+    "items_opt": null,
+    "attr": {
+        "title": "Megaman X8 16-bit",
+        "porter": [
+            "Damian2809"
+        ],
+        "desc": "The Mega Man X8 16-Bit Demake is a fan-made project that reimagines the original Mega Man X8 in the style of the classic 16-bit era. This demake brings the modern aesthetics and mechanics of the original game into a retro environment reminiscent of the SNES era, particularly the earlier Mega Man X titles on that system.",
+        "desc_md": null,
+        "inst": "https://sonicfangameshq.com/forums/showcase/mega-man-x8-16-bit.2184/?sfnsn=wa grab the zip from here, extract it and put the files in the megamanx816-bit/gamedata folder",
```

Comment:
Is there no better URL for this? What if the forum goes down? You can also shorten it to `https://sonicfangameshq.com/forums/showcase/mega-man-x8-16-bit.2184` and it'll work fine.